src/test/scala/scatan/utils/UnorderedTupleTest.scala

```diff
@@ -0,0 +1,79 @@
+package scatan.utils
+
+import cats.kernel.Monoid
+import cats.syntax.semigroup.*
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import scatan.BaseTest
+import scatan.model.map.Hexagon.given
+import scatan.model.map.HexTiledMap.*
+import scatan.utils.{UnorderedPair, UnorderedTriple}
+
+class UnorderedPairTest extends BaseTest with ScalaCheckPropertyChecks:
+
+  "A UnorderedPair" should "have 2 elements" in {
+    assertCompiles("UnorderedPair(0, 0)")
+    assertDoesNotCompile("UnorderedPair()")
+  }
+
+  it should "be equals based on its elements" in {
+    forAll { (a: Int, b: Int) =>
+      val pair1 = UnorderedPair(a, b)
+      val pair2 = UnorderedPair(a, b)
+      pair1 shouldBe pair2
```

Comment:
An example of property check
```suggestion
    forAll { (a: Int, b: Int) =>
      val pair1 = UnorderedPair(a, b)
      val pair2 = UnorderedPair(a, b)
      pair1 shouldBe pair2
```
e.g. `a` & `b` get some random number during tests